src/app/classes/Container.js

```diff
@@ -2,9 +2,32 @@ import Row from './Row'
 
 export default class Container {
   constructor(items = []) {
+    let status;
+    do {
+      items.sort((a, b) => a.top - b.top);
+      items.sort((a, b) => a.left - b.left);
```

Comment:
This will overwrite the previous line's sort. If you want this as a secondary sort you would need to do something like

`items.sort((a, b) => (a.top - b.top) * 1000 + a.left - b.left);`